spec/factories/donation_site.rb

```diff
@@ -3,5 +3,8 @@
     organization { Organization.try(:first) || create(:organization) }
     name { "Smithsonian Conservation Center" }
     address { "1500 Remount Road, Front Royal, VA 22630" }
+    contact_name { nil }
```

Comment:
I believe this is redundant. If you don't provide the attributes, it will be nil. 

You could use a Factory bot sequence to provide it with data. This could look like
```
 sequence(:phone) { |n| "555-555-%04d" % n }
```

Reply:
@doliveirakn I opted to use Faker to do this. LMK if there is anything I could improve, the Address with faker generated some issues, so I decided to stay with the current one.

Reply:
Faker is a good way to do it 👍